src/controllers/informationController.js

```diff
@@ -0,0 +1,56 @@
+const mongoose = require('mongoose');
+// const userProfile = require('../models/userProfile');
+// const hasPermission = require('../utilities/permissions');
+const escapeRegex = require('../utilities/escapeRegex');
+
+
+const informationController = function (Information) {
+  const getInformations = function (req, res) {
+    Information.find({}, 'infoName infoContent')
+      .then(results => res.status(200).send(results))
+      .catch(error => res.status(404).send(error));
+  };
+
+
+  const addInformation = function (req, res) {
```

Comment:
You first create a new Information and then create another new Information after checking for name uniqueness.  I suggest you may first check for name uniqueness, and then create the Information.